.github/workflows/rebuild-released-images.yaml

```diff
@@ -84,23 +84,25 @@ jobs:
         shell: bash
         run: |
           go mod download
-      - name: Build and push operator to the DockerHub (daily-tag)
-        uses: docker/build-push-action@v3
+      - name: Build and push operator to Dockerhub & Quay (daily-tag)
+        uses: ./.github/actions/build-push-image
```

Comment:
I'm curious if this is going to be executed from the "main" branch, not from the one that we checked out code (actions/checkout) from in the previous step?